zubhub_frontend/zubhub/src/components/form/tagsInput/TagsInput.jsx

```diff
@@ -70,20 +112,20 @@ export default function TagsInput({
     </CustomButton>
   ));
 
-  selectedTags = selectedTags?.map((tag, index) => (
+  const selectedTagsRender = selectedTags?.map((tag, index) => (
```

Comment:
consider returning a memorized version of this

Reply:
Hi, 

Please can you give me more highlights of this?

Reply:
const selectedTagsRender = useMemo(() => {
    return selectedTags?.map((tag, index) => {
        // JSX rendering logic here
    });
}, [selectedTags, /* dependencies */]);

It's ideally a best practice `perfomance wise` to prevent unnecessary re-renders 
Checkout [here](https://www.linkedin.com/pulse/avoid-unnecessary-re-renders-reactjs-christian-zanchetta) for more